packages/airnode-deployer/src/cli/index.ts

```diff
@@ -91,95 +86,76 @@ yargs(hideBin(process.argv))
     }
   )
   .command(
-    'remove',
-    'Removes a deployed Airnode instance',
+    'remove-with-receipt',
+    'Removes a deployed Airnode instance using a receipt file',
     {
       receipt: {
         alias: 'r',
         description: 'Path to receipt file',
+        default: 'config/receipt.json',
         type: 'string',
       },
+    },
+    async (args) => {
+      logger.debugMode(args.debug as boolean);
+      logger.debug(`Running command ${args._[0]} with arguments ${longArguments(args)}`);
+
+      await runCommand(() => removeWithReceipt(args.receipt));
+      return;
+    }
+  )
+  .command(
+    'remove-with-deployment-details',
+    'Removes a deployed Airnode instance using the Airnode short address and cloud provider specifications',
+    {
       'airnode-address-short': {
         alias: 'a',
         description: 'Airnode Address (short version)',
         type: 'string',
+        demandOption: true,
       },
       stage: {
         alias: 's',
         description: 'Stage (environment)',
         type: 'string',
+        demandOption: true,
       },
       'cloud-provider': {
         alias: 'c',
         description: 'Cloud provider',
         choices: ['aws', 'gcp'] as const,
+        demandOption: true,
       },
       region: {
         alias: 'e',
         description: 'Region',
         type: 'string',
+        demandOption: true,
       },
       'project-id': {
         alias: 'p',
-        description: 'Project ID (GCP only)',
+        description: 'Project ID (required for GCP only)',
         type: 'string',
       },
     },
     async (args) => {
       logger.debugMode(args.debug as boolean);
       logger.debug(`Running command ${args._[0]} with arguments ${longArguments(args)}`);
-      const receiptRemove = !!args.receipt;
-      const descriptiveArgsCommon = ['airnode-address-short', 'stage', 'cloud-provider', 'region'];
-      const descriptiveArgsCloud = {
-        aws: descriptiveArgsCommon,
-        gcp: [...descriptiveArgsCommon, 'project-id'],
-      };
-      const descriptiveArgsAll = uniq(
-        Object.values(descriptiveArgsCloud).reduce((result, array) => [...result, ...array])
-      );
-      const argsProvided = intersection([...descriptiveArgsAll, 'receipt'], keys(args));
-      const descriptiveArgsProvided = intersection(descriptiveArgsAll, keys(args));
 
-      if (isEmpty(argsProvided)) {
-        // Throwing strings to prevent yargs from showing error stack trace
```

Comment:
Can you keep this comment on the throw statement?

Reply:
Yes- comment restored.